core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionNetworkOutput.java

```diff
@@ -33,294 +34,198 @@
 import org.apache.http.client.methods.HttpPost;
 import org.apache.http.conn.ConnectionPoolTimeoutException;
 import org.apache.http.entity.ByteArrayEntity;
+import org.apache.http.util.EntityUtils;
+import org.apache.commons.lang3.exception.ExceptionUtils;
 
-import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
 import java.net.SocketException;
 import java.net.UnknownHostException;
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.TimeZone;
 import java.util.concurrent.TimeUnit;
 
 /**
- * The class is responsible for the actual sending of {@link com.microsoft.applicationinsights.internal.channel.common.Transmission}
+ * The class is responsible for the actual sending of
+ * {@link com.microsoft.applicationinsights.internal.channel.common.Transmission}
  *
  * The class uses Apache's HttpClient framework for that.
  *
  * Created by gupele on 12/18/2014.
  */
 public final class TransmissionNetworkOutput implements TransmissionOutput {
```

Comment:
There are bunch of methods in this class which do not have java docs. It would be good if we can update those places as we are changing this part.

Reply:
Added docs for public and private methods.